src/routes/users_routes.js

```diff
@@ -408,6 +408,17 @@ router.get('/login', (req, res) => {
   return render(req, res, 'LoginPage');
 });
 
+const getLoginRedirect = (req) => {
```

Comment:
Unrelated to User typescript conversion, but I noticed the weirdness of logging in and being redirected to login. whoops